iac/terraform/aws/web-client/route53.tf

```diff
@@ -0,0 +1,22 @@
+locals {
+  web-client-domain = var.environment == "master" ? var.bloodconnect_domain : "${var.environment}.${var.bloodconnect_domain}"
+}
+
+data "aws_route53_zone" "main" {
+  #checkov:skip=CKV2_AWS_39: "Ensure Domain Name System (DNS) query logging is enabled for Amazon Route 53 hosted zones"
```

Comment:
i believe we have a ticket for this, if so let's mention in pr description.

Reply:
No, we don't have any ticket with this. I think you are talking about DNSSEC.

Reply:
Why would we need to have DNS query logging separately?